core/src/main/java/org/apache/carbondata/common/iudprocessor/cache/BlockletLevelDeleteDeltaDataCache.java

```diff
@@ -0,0 +1,29 @@
+package org.apache.carbondata.common.iudprocessor.cache;
+
+import org.roaringbitmap.RoaringBitmap;
+
+/**
+ * Created by S71955 on 06-10-2016.
+ */
+public class BlockletLevelDeleteDeltaDataCache {
```

Comment:
As an opinion I think it would be better if we can rename the class to BlockletLevelDeleteDeltaCache